core/database/foxx/api/user_router.js

```diff
@@ -879,8 +1822,52 @@ router
             delete user.refresh;
 
             res.send([user]);
+            console.info(
+                "Client:",
+                client?._id || "unknown",
+                "|",
+                "Correlation_ID:",
+                req.headers["x-correlation-id"],
+                "|",
+                "HTTP:",
+                "GET",
+                "|",
+                "Route:",
+                "/view",
+                "|",
+                "Status:",
+                "Success",
+                "|",
+                "Desc:",
+                "View User Information",
```

Comment:
It would be more useful if in the description if something about this was included. If the setting is turned on or off.

queryParam("details", joi.boolean().optional(), "Show additional user details")